<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsdStageNodeAnimationRollup</class>
 <widget class="QWidget" name="UsdStageNodeAnimationRollup">
  <property name="windowTitle">
   <string>Animation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="PlaybackTypeGroupBox">
     <property name="toolTip">
      <string>Use Playback Type settings to configure the animation of your USD stage. 
You can specify an original range, a custom range, a custom start 
speed, or specific USD timeCode.</string>
     </property>
     <property name="title">
      <string>Playback Type</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_1">
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="AnimationMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="StartFrameLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start Frame</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="MaxSDK::QmaxDoubleSpinBox" name="CustomAnimationStartFrame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>-9999999</number>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="EndFrameLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>End Frame</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="MaxSDK::QmaxDoubleSpinBox" name="CustomAnimationEndFrame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>-9999999</number>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="SpeedLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Speed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="MaxSDK::QmaxDoubleSpinBox" name="CustomAnimationSpeed">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>-9999999</number>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="UsdTimecodeLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>USD Time</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="MaxSDK::QmaxDoubleSpinBox" name="CustomAnimationPlaybackTimecode">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="ClampFramesLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Clamp:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="ClampFrames">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="MaxAnimationDataGroupBox">
     <property name="toolTip">
      <string>3ds Max Animation Data displays the result of converting the animation to 3ds Max data, 
matching the frames per second set in your 3ds Max scene, and applying playback settings.</string>
     </property>
     <property name="title">
      <string>3ds Max Animation Data</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="MaxStartFrameLabel">
        <property name="text">
         <string>New Start:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="MaxAnimationStartFrame"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="MaxEndFrameLabel">
        <property name="text">
         <string>New End:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="MaxAnimationEndFrame"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="MaxFPSLabel">
        <property name="text">
         <string>FPS:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="MaxAnimationFPS"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="SourceAnimationDataGroupBox">
     <property name="toolTip">
      <string>Source Info displays the animation timeCode information found in
your USD animation (Start, End, and TimeCodes Per Second).</string>
     </property>
     <property name="title">
      <string>Source Info (USD TimeCode)</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="USDStartTimeCodeLabel">
        <property name="text">
         <string>Start:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="SourceAnimationStartTimeCode"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="USDEndTimeCodeLabel">
        <property name="text">
         <string>End:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="SourceAnimationEndTimeCode"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="SourceTPSLabel">
        <property name="text">
         <string>TPS:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="SourceAnimationTPS"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaxSDK::QmaxDoubleSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">Qt/QmaxSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
